src/ref.builds

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="*\ref\**\*.*proj">
+    <Project Include="*\ref\**\*.builds;*\ref\**\*.*proj">
```

Comment:
I believe that with this approach some projects will get built twice? wherever there is a builds file, we shouldn't build the csprojs that it calls, but I'm not sure how to do this. @ericstj thoughts?